script.js

```diff
@@ -1,14 +1,222 @@
+    popupProject.classList.toggle('popup');
+    popupProject.innerHTML = `<div class="pop-up-project-block">
+      <div class="desk-pop-b-one">
+        <div class="pop-title-block">
+          <h2  class="contact-title font-montesori">${table[index].name}</h2>
+          <button  class="close-pop-up" type="button">
+            <img src="assets/icons/Icon - Cancel.png" alt="" class="cancel-popup">
+          </button>
+        </div>
+        <ul class="pop-up-tools tools-container">
+          <li class="tools">${table[index].technologies[0]}</li>
+          <li class="tools">${table[index].technologies[1]}</li>
+          <li class="tools">${table[index].technologies[2]}</li>
+        </ul>
+      </div>
+      <div class="desk-pop-b-one-x">
+
+      <div class="desk-pop-b-two">
+        <img src="${table[index].featuredImage}" alt=""  class="project-popup-image">
+      </div>
+  
+     <div class="desk-pop-b-three">
+      <p class="pop-up-text font-montesori blue-2"  >${table[index].description}</p>
+      <div class="pop-up-btn">
+        <button class="see-live-btn see-project-btn"><a href="${recent[0].linkToLive_version}"></a>See Live<img src="assets/icons/Icon.png" alt=""></button>
+        <button class="see-source-btn see-project-btn"><a href="${recent[0].linkToSource}"></a>See Source<img src="assets/icons/1.png" alt=""></button>
+      </div>
+     </div>
+     </div>
+
+    </div>`;
+
+    document.querySelector('.close-pop-up').addEventListener('click', () => {
+      popupProject.classList.remove('popup');
+    });
+  });
+});
+
+const recentProjectPopup = document.querySelector('.see-project-btn');
+
+recentProjectPopup.addEventListener('click', () => {
+  popupProject.classList.toggle('popup');
+  popupProject.innerHTML = `<div class="pop-up-project-block">
+   <div class="desk-pop-b-one">
+     <div class="pop-title-block">
+       <h2  class="contact-title font-montesori">${recent[0].name}</h2>
+       <button  class="close-pop-up" type="button">
+         <img src="assets/icons/Icon - Cancel.png" alt="" class="cancel-popup">
+       </button>
+     </div>
+     <ul class="pop-up-tools tools-container">
+       <li class="tools">${recent[0].technologies[0]}</li>
+       <li class="tools">${recent[0].technologies[1]}</li>
+       <li class="tools">${recent[0].technologies[2]}</li>
+     </ul>
+   </div>
+<div class="desk-pop-b-one-x">
+   <div class="desk-pop-b-two">
+     <img src="${recent[0].featuredImage}" alt="" class="project-popup-image">
+   </div>
+
+  <div class="desk-pop-b-three">
+   <p class="pop-up-text font-montesori blue-2"  >${recent[0].description}</p>
+   <div class="pop-up-btn">
+     <button class="see-live-btn see-project-btn"><a href="${recent[0].linkToLive_version}"></a>See Live<img src="assets/icons/Icon.png" alt=""></button>
+     <button class="see-source-btn see-project-btn"><a href="${recent[0].linkToSource}"></a>See Source <img src="assets/icons/1.png" alt=""></button>
```

Comment:
[OPTIONAL] Kindly make sure these two buttons and the image are aligned well to the bottom as in the Figma template.
<img width="1440" alt="Screenshot 2022-12-23 at 8 36 52 PM" src="https://user-images.githubusercontent.com/25887253/209383459-5e80c109-a30c-4559-a99e-a145aa7cf271.png">